configure/CONFIG_COMMON

```diff
@@ -66,6 +66,7 @@ INSTALL_HOST_LIB            = $(INSTALL_LOCATION_LIB)/$(EPICS_HOST_ARCH)
 
 INSTALL_INCLUDE             = $(INSTALL_LOCATION)/include
 INSTALL_DOC                 = $(INSTALL_LOCATION)/doc
+INSTALL_MARKDOWN            = $(TOP)/documentation/generated
```

Comment:
Maybe @minijackson could comment on this? I do not remember exactly but I thought this target was for RTD build only and would not be installed in a normal build. I think that RTD expects to find the markdown files in the source tree; maybe there is a way to work around this, if necessary. 
For RTD builds, markdown files are the sources ;-)

Reply:
I agree with @ttkorhonen, I think the reason I chose TOP and not INSTALL_LOCATION, is that the markdown files are not that useful as installed documentation, and would make the build more complicated when we need to run readthedocs afterwards.

Reply:
I've changed the naming, but I'm not sure what to do about the `clean` / `distclean`, since the `documentation` is not included via the `DIRS` variable. I've had issues trying that before. I'm not sure this is a big issue, since the documentation build is not run by default.